scripts/train_agents.py

```diff
@@ -464,20 +79,40 @@ def N_X_SP(args) -> None:
         unseen_teammates_len=unseen_teammates_len,
         )
 
+def SPN_1ADV(args) -> None:
```

Comment:
Can you add a comment under here for the description you sent on slack?
"n-1-sp with a single adv teammate"